types/react-native/Libraries/Components/TextInput/InputAccessoryView.d.ts

```diff
@@ -14,7 +14,7 @@ export class InputAccessoryView extends React.Component<InputAccessoryViewProps>
 export interface InputAccessoryViewProps {
   backgroundColor?: ColorValue | undefined;
 
-  children?: React.ReactNode;
+  children?: React.ReactNode | undefined;
```

Comment:
`undefined` is already in `ReactNode` so this is redundant.

Reply:
I think this was done by an ESLint plug-in ensuring compatibility with “ exactOptionalPropertyTypes”. The backing Flow types do not have a similar concept, so for consistency we enforce optional properties always can accept undefined values.

I think it’s just a simple AST plug-in that doesn’t follow the backing types.